python/declarative/models/project.py

```diff
@@ -18,7 +18,7 @@ def parse_dates():
         raise NotImplementedError
 
     nlihc_id = Column(        
-        "nlihc_id",
+        "Nlihc_id",
```

Comment:
Hey @ohaaga per your commit comment and this code, does this mean you're making the column name in the CSV file always match the column name in SQL? We want to be able to have those be different, actually. Being able to specify a different name solves a few problems for us:
- running SQL queries on fields with capital letters, spaces, etc. gets much more complicated - SQL allows you to create fields with names with non-standard characters but then requires you to always quote the field name in your query. This makes querying the database more difficult
- Sometimes the field name is not descriptive so we want a better name in our database
- Sometimes the same type of field is called one thing in one data source and something else in another - we want to be able to use the same name for all tables in our database. For example, in some data sets the neighborhood cluster is called 'neighborhoodcluster' and in others it is 'neighborhood_cluster' - we want all tables to use a consistent approach in our database.